controls/auth.js

```diff
@@ -78,7 +79,7 @@ exports.postLogin = function (req, res) {
 */
 exports.getLogout = function (req, res) {
```

Comment:
Try getting the current url in the `req` param and then redirect back to that url after calling the logout function.